wd_rsa.c

```diff
@@ -167,6 +167,12 @@ static int wd_rsa_common_init(struct wd_ctx_config *config, struct wd_sched *sch
 
 static int wd_rsa_common_uninit(void)
 {
+	enum wd_status status;
+
+	wd_alg_get_init(&wd_rsa_setting.status, &status);
+	if (status == WD_UNINIT)
+		return -WD_EINVAL;
+
```

Comment:
Uninit for RSA algorithm was added. Then why not add it for DH and ECC algorithms?

Reply:
Let’s fix the problem that priv space needs to be reallocated in this patch：https://github.com/Linaro/uadk/pull/612